var/spack/repos/builtin/packages/gaudi/package.py

```diff
@@ -23,25 +29,40 @@ class Gaudi(CMakePackage):
     variant('vtune', default=False,
             description='Build with Intel VTune profiler support')
 
+    # only build subdirectory GaudiExamples when +tests
+    patch("build_testing.patch", when="@:33.1")
+    # fix for the new cmake config, should be merged in branch
+    patch('python2.patch', when="@develop")
+    # fixes for the cmake config which could not find newer boost versions
+    patch("link_target_fixes.patch", when="@33.0:33.1")
+    patch("link_target_fixes32.patch", when="@:32.2")
```

Comment:
Have these patches been submitted upstream?

Reply:
The `python2.patch`, yes. The others become obsolete with v34 and I'm not sure there will be another v33 patch release. In any case, the developers are aware of the issue and will include it if there is another release before the cmake modernisation.